viewer/packages/utilities/src/events/InputHandler.ts

```diff
@@ -9,6 +9,9 @@ import { Vector2 } from 'three';
 
 type PointerEventDelegate = (event: { offsetX: number; offsetY: number }) => void;
 
+//TODO savokr 17-01-2022: Change EventCollection type to be more specific by contstraining types of possible events (see draft PR).
+export type EventCollection = { [eventName: string]: EventTrigger<(...args: any[]) => void> };
```

Comment:
```suggestion
export type EventCollection = { [eventName: string]: EventTrigger<(...args: any[]) => void> };
```
This was actually better than I thought - I think this solution is ok. My draft PR maybe improves a tiny bit, but nothing substantial.